reference/5.1/Microsoft.PowerShell.Core/About/about_Member-Access_Enumeration.md

```diff
@@ -0,0 +1,303 @@
+collection and attempts to access the specified member on each item.
+
+Member-access enumeration helps you write simpler and shorter code. Instead of
+piping a collection object to `ForEach-Object` or using the `ForEach()`
+[intrinsic method](about_Intrinsic_Members.md#methods) to access members on
+each item in the collection, you can use the member-access operator on the
+collection object.
+
+These commands are functionally identical:
+
+```powershell
+Get-Service -Name event* | ForEach-Object -Process { $_.DisplayName }
+(Get-Service -Name event*).ForEach({ $_.DisplayName })
+(Get-Service -Name event*).DisplayName
+```
+
+```Output
+Windows Event Log
+COM+ Event System
+
+Windows Event Log
+COM+ Event System
+
+Windows Event Log
+COM+ Event System
+```
+
+> [!NOTE]
+> You can use the member-access operator to get the values of a property on
+> items in a collection but you can't use it to set them directly. For more
+> information, see [about_Arrays](about_Arrays.md#member-access-enumeration).
+
+When you use the member-access operator on any object and the specified member
+exists on that object, the member is invoked. For property members, the operator
+returns the value of that property. For method members, the operator calls that
+method on the object.
+
+When you use the member-access operator on a list collection object that doesn't
+have the specified member, PowerShell automatically enumerates the items in
+that collection and uses the member-access operator on each enumerated item.
+
+You can check if an object is a list collection by seeing whether its type
+implements the **IList** interface:
+
+```powershell
+$List = @('a', 'b')
+$Hash = @{ a = 'b' }
+$List.GetType().ImplementedInterfaces.Name -contains 'IList'
+$Hash.GetType().ImplementedInterfaces.Name -contains 'IList'
+```
+
+```Output
+True
+
+False
+```
+
+During member-access enumeration for a property, the operator returns the value
+of the property for each item that has that property. If no items have the
+specified property, the operator returns `$null`.
+
+During member-access enumeration for a method, the operator attempts to call the
+method on each item in the collection. If any item in the collection does does
+not have the specified method, the operator returns the **MethodNotFound**
+exception.
+
+> [!WARNING]
+> During member-access enumeration for a method, the method is called on each
+> item in the collection. If the method you are calling makes changes, the
+> changes are made for every item in the collection. If an error occurs during
+> enumeration, the method is called only on the items enumerated before the
+> error. For additional safety, consider manually enumerating the items and
+> explicitly handling any errors.
+
+The following examples detail the behavior of the member-access operator under
+all possible scenarios.
+
+## Accessing members of a non-list object
+
+When you use the member-access operator on an object that is not a collection
```

Comment:
Shouldn't this be `list collection` than `collection list`?

Reply:
Fixed